docs/src/app/components/pages/components/RadioButton/ExampleSimple.jsx

```diff
@@ -30,6 +32,13 @@ const RadioButtonExampleSimple = () => (
         disabled={true}
         style={styles.radioButton}
       />
+      <RadioButton
+        value="ludicrous"
```

Comment:
You need to change the value:
![capture d ecran 2016-02-10 a 20 15 08](https://cloud.githubusercontent.com/assets/3165635/12958652/fe72b3f0-d032-11e5-8a65-2b4a24ad660b.png)